doc/api/crypto.md

```diff
@@ -1585,7 +1585,8 @@ behavior.
 
 The `algorithm` is dependent on the available algorithms supported by the
 version of OpenSSL on the platform. Examples are `'sha256'`, `'sha512'`, etc.
-On recent releases of OpenSSL, `openssl list-message-digest-algorithms` will
+On recent releases of OpenSSL, `openssl list -digest-algorithms` ( 
+`openssl list-message-digest-algorithms` for older versions of OpenSSL ) will
```

Comment:
Would you be so kind and remove the whitespace in the beginning and end of the bracket? And move the opening bracket to the second line, so it is nicer to read.

Reply:
Done @BridgeAR.